src/tests/Microsoft.PowerFx.Interpreter.Tests/MutationFunctionsTests.cs

```diff
@@ -380,5 +381,21 @@ public override bool TryShallowCopy(out FormulaValue copy)
                 return true;
             }
         }
+
+        [Theory]
+        [InlineData("x=1;y=2;z=x+y;", "Abs(-z)", 3)]
+        public void NameFormulasTest(string namedFormulas, string expression, decimal expected)
```

Comment:
should this be in MutationFunctionsTests? Doesn't appear to do mutation. A new file of tests for UserDefinitions might be good here

Reply:
Temporary placement. It will be moved.

Reply:
nit: missing the D in the method name "NamedFormulasTest"